refactored counter_test to use assert statements and renamed variable…

conwaychriscosmo opened the pull request:
…s for consistency

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
5. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

/kind cleanup

**What this PR does / why we need it**:
The PR refactors a test to be more readable.  It is unnecessary but nice.

**Special notes for your reviewer**:
It is my first code change.
**Does this PR introduce a user-facing change?**:
<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

conwaychriscosmo committed:
refactored counter_test to use assert statements and renamed variables for consistency

conwaychriscosmo commented:
@mtaufen @RainbowMango @logicalhan @yastij Please let me know your thoughts and approve my PR.

conwaychriscosmo commented:
\wg component-standard

RainbowMango commented on staging/src/k8s.io/component-base/metrics/counter_test.go:
```diff
@@ -19,6 +19,7 @@ package metrics
 import (
 	"bytes"
 	"testing"
+	"github.com/stretchr/testify/assert"
```
nit: Imports should be grouped. standard libs -> third party -> k8s imports

yastij replied:
+1

mtaufen replied:
@conwaychriscosmo, as @RainbowMango mentioned it's conventional in the K8s project to separate each group of imports with a blank line. In this case, the assert lib should be in the middle group, along with semver and expfmt.

RainbowMango commented:
@conwaychriscosmo 
Thanks for your contribution.

> The PR refactors a test to be more readable. It is unnecessary but nice.

Would you mind introduce `github.com/stretchr/testify/assert` to all of the tests under this package?

yastij reviewed (commented):
mostly lgtm, one nit on the imports
/ok-to-test

conwaychriscosmo commented:
> @conwaychriscosmo
> Thanks for your contribution.
> 
> > The PR refactors a test to be more readable. It is unnecessary but nice.
> 
> Would you mind introduce `github.com/stretchr/testify/assert` to all of the tests under this package?

I was hoping that people would want that :)

fedebongio commented:
/assign @logicalhan

mtaufen reviewed (commented):
Looks correct to me.

Please run gofmt on your changes.

RainbowMango commented:
> Looks correct to me.
> 
> Please run gofmt on your changes.

Yes, Run `./hack/update-gofmt.sh` in the kubernetes directory.

conwaychriscosmo commented:
/test pull-kubernetes-kubemark-e2e-gce-big
/test pull-kubernetes-e2e-gce-100-performance

RainbowMango commented:
@conwaychriscosmo 
Would you mind fix the comments and squash your commits?

conwaychriscosmo commented:
@RainbowMango 
My mistake not properly addressing the comments, moving an import statement two lines isn't hard; I think I accidentally reverted the changes that addressed them when I amended the commit with a new author. I would rather not squash the commits because the documentation I read about testing and contributing said squashing commits was optional.

logicalhan commented:
> I would rather not squash the commits because the documentation I read about testing and contributing said squashing commits was optional.

I would also prefer that you squash the commits.

conwaychriscosmo commented:
/remove area/apiserver

mtaufen commented:
/lgtm

logicalhan reviewed (approved):
/lgtm

logicalhan commented:
/approve
